mesonbuild/mesondata.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Meson development team
+# Copyright 2021 The Meson development team
```

Comment:
Shouldn't this be `2020-2021`?

Reply:
Not according to `gen_data.py` :p

But, really I'd think the copyright info is invalid and meaningless, generated files don't have license info -- only the source files do (or do not actually, in this case).